src/containers/tutor-home-page/subjects-step/SubjectsStep.jsx

```
@@ -92,24 +157,31 @@
 
-  const handleCategoryChange = useCallback((newValue) => {
-    console.log('handleCategoryChange called with:', newValue)
-    if (!newValue || !newValue.value) {
-      setSelectedCategory(null)
-      setSelectedSubject(null)
-      return
-    }
-    setSelectedCategory((prev) =>
-      prev?.value === newValue.value ? prev : newValue
-    )
+const handleCategoryChange = useCallback((newValue) => {
```

ESLint

[error] Replace `const·handleCategoryChange·=·useCallback(` with `··const·handleCategoryChange·=·useCallback(⏎····`

(prettier/prettier)